lib/samples/add_raster_from_service/add_raster_from_service.dart

```diff
@@ -0,0 +1,105 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:flutter/material.dart';
+
+class AddRasterFromService extends StatefulWidget {
+  const AddRasterFromService({super.key});
+
+  @override
+  State<AddRasterFromService> createState() => _AddRasterFromServiceState();
+}
+
+class _AddRasterFromServiceState extends State<AddRasterFromService> {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
```

Comment:
Given that there are no buttons and no onTap actions, this can be pared back. We can remove _ready, SafeArea/Stack/Column/Expanded, the progress indicator, and the Future/async on the onMapViewReady declaration. See for example, `add_tiled_layer.dart`.